test/ClangImporter/availability_non_runtime_protocol_objc_protocol.swift

```diff
@@ -0,0 +1,27 @@
+	
+// REQUIRES: objc_interop
+// REQUIRES: foundation
+
+// RUN: %target-swift-frontend -enable-objc-interop -typecheck -verify  -import-objc-header %S/Inputs/availability_non_runtime_protocol_objc_protocol.h %s -emit-objc-header -emit-objc-header-path %t/SwiftCallObjC-Swift.h 
+
+import Foundation
+
+class AppDelegate {
+    @inline(never) func FOOBARwrapper() {
+        // CustomObject is defined in ObjC.
+        let instanceOfCustomObject = CustomObject()
+        instanceOfCustomObject.someProperty = "Hello World"
+        print(instanceOfCustomObject.someProperty)
+        instanceOfCustomObject.someMethod()
+    }
+}
```

Comment:
This code looks like it might be superfluous to what is being tested in the Swift compiler here